### PR TITLE
Add the organization worklow documentation

### LINKED ISSUE #15: Document poll creation and general admin workflow

### PR DESCRIPTION
**Pull request by m09**
Fixes #15.

### CODE REVIEW

**Comment by bzz** [comment]
Looks good, but what do you think about also linking it from the main README.md ?

Similar to how READMEs of the projects contain user and developer facing instructions, how about adding a new section called something like "Organizing" and explaining in there that if you are an organizers, check that guide out?

File: organization-workflow.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    When we have time we link the authors to their respective
+    homepages and include relevant links (code, data, etc).
+
+- PR a change to the next session in the
+  [readme](README.md#next-sessions).
+
+    We use the following template:
+
+        - __YYYY.MM.DD__ at __4PM CET__: [Title of the paper](url of the pdf on GDrive). We take notes in a [public GDoc](url of the notes on GDrive), you are very welcome to ask questions or give thoughts in it!
```
Comment (bzz) [inline organization-workflow.md:55]: Usually, as soon as winner is picked (or PR is merged) - we also announce that on Slack #reading-club

**Comment by m09** [comment]
@bzz I addressed your points if you want to give it another look :)

File: organization-workflow.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Create a poll to decide on the next paper.
+
+The poll should be created on the #reading-club community channel. It
+uses the poll widget:
+
+    /poll "What papers would you like to study during next session, on the xxth of xxxxx?" "Title1: url1" "Title2: url2"
+
+## 5 days before
+
+- Pick the winner (most votes) and announce it in the community slack.
```
Comment (bzz) [inline organization-workflow.md]:
```suggestion
- Pick the winner (most votes) and announce it on [community slack #reading-club channel](https://sourced-community.slack.com/messages/CCLPL1WQH).
```

File: organization-workflow.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# Organization workflow
+
+The general workflow to organize a session is as follows:
```
Comment (bzz) [inline organization-workflow.md]:
```suggestion
The general workflow to organize a reading club session is as follows:
```

File: organization-workflow.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+- Create the next paper candidates issue.
+
+    The corresponding issue template is almost good to go: it just
+    requires setting the date of the next session in the title.
+
+- Archive last session and create the new one.
+
+    PR a change to
+    [src-d/reading](https://github.com/src-d/reading-club) club to put
```
Comment (bzz) [inline organization-workflow.md]:
```suggestion
    [src-d/reading-club](https://github.com/src-d/reading-club) to put
```

File: organization-workflow.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    
+    And create the new session with the template
+
+        - __YYYY.MM.DD__ at __4PM CET__: [TBD](url of the candidates issues)
+
+## 1 week before
+
+Create a poll to decide on the next paper.
+
+The poll should be created on the #reading-club community channel. It
```
Comment (bzz) [inline organization-workflow.md]:
```suggesion
The poll should be created on the [#reading-club community Slack channel](https://sourced-community.slack.com/messages/CCLPL1WQH). It
```

File: organization-workflow.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+uses the poll widget:
+
+    /poll "What papers would you like to study during next session, on the xxth of xxxxx?" "Title1: url1" "Title2: url2"
+
+## 5 days before
+
+- Pick the winner (most votes) and announce it in the community slack.
+
+- Add the PDF to the [GDrive
```
Comment (bzz) [inline organization-workflow.md]:
```suggestion
- Add the PDF of the paper to the [GDrive
```

File: organization-workflow.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+  directory](https://drive.google.com/drive/u/1/folders/1cAzkDW_sXb49gRZvvhVvXu3Wi_84lQdn).
+
+- Create the notes document.
```
Comment (bzz) [inline organization-workflow.md]:
```suggestion
- Create a new notes document in the [`Notes` dir on GDrive](https://drive.google.com/drive/folders/1IQ7-wOea32lDkd69-NcLw5xYB74oH6rJ).
```

File: organization-workflow.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+    When we have time we link the authors to their respective
+    homepages and include relevant links (code, data, etc).
+
+- PR a change to the next session in the
```
Comment (bzz) [inline organization-workflow.md]:
```suggestion
- PR a change with the next session to this repository's
```

File: organization-workflow.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+Announce the session on the community slack.
+
+## One hour before
+
+Announce the session on the community slack again.
+
+## Ten minutes before
+
+- Launch the zoom meeting.
```
Comment (bzz) [inline organization-workflow.md]:
```suggestion
- Launch the [zoom meeting](https://github.com/src-d/reading-club#where).
```

File: organization-workflow.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+- Announce the session on the community slack again with the Zoom ID.
```
Comment (bzz) [inline organization-workflow.md]:
```suggestion
- Announce the session on the community slack again with the Zoom meeting ID.
```

**Review by bzz — APPROVED** [review]
👍 

sans few minor suggestions below

**Pull request merged**